actionmailbox/lib/action_mailbox.rb

```diff
@@ -14,4 +14,5 @@ module ActionMailbox
   mattr_accessor :incinerate, default: true
   mattr_accessor :incinerate_after, default: 30.days
   mattr_accessor :queues, default: {}
+  mattr_accessor :amazon, default: {}
```

Comment:
Would move this config over to the ingress controller. `ActionMailbox.amazon` is not something to expose for general usage.